src/registrar/admin.py

```diff
@@ -187,43 +200,48 @@ class MyUserAdmin(BaseUserAdmin):
         "email",
         "Permissions",
         "is_active",
-        "is_staff",
-        "is_superuser",
+        "groups",
```

Comment:
Question: since I think we still have a is_staff variable, is that editable or is it not because of hierarchy (groups contains is_staff which means its read_only)?

Reply:
It is editable but I no longer think an analyst needs or should have eyes on, since permissions and roles are managed through groups.